## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 505702`
Commit: `31771fcec3f8669c8845e2c6b098779ab55c5daa`

---
### `x86_64-linux`
<details>
  <summary>:x: 12 packages failed to build:</summary>
  <ul>
    <li>aider-chat-full</li>
    <li>aider-chat-full.dist</li>
    <li>aider-chat-with-help</li>
    <li>aider-chat-with-help.dist</li>
    <li>python313Packages.momepy</li>
    <li>python313Packages.momepy.dist</li>
    <li>python313Packages.pandera</li>
    <li>python313Packages.pandera.dist</li>
    <li>python314Packages.momepy</li>
    <li>python314Packages.momepy.dist</li>
    <li>python314Packages.pandera</li>
    <li>python314Packages.pandera.dist</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 235 packages built:</summary>
  <ul>
    <li>alpaca</li>
    <li>libretranslate (python313Packages.libretranslate)</li>
    <li>libretranslate.dist (python313Packages.libretranslate.dist)</li>
    <li>newelle</li>
    <li>overturemaps</li>
    <li>overturemaps.dist</li>
    <li>pyspread</li>
    <li>pyspread.dist</li>
    <li>python313Packages.argos-translate-files</li>
    <li>python313Packages.argos-translate-files.dist</li>
    <li>python313Packages.argostranslate</li>
    <li>python313Packages.argostranslate.dist</li>
    <li>python313Packages.arviz</li>
    <li>python313Packages.arviz-plots</li>
    <li>python313Packages.arviz-plots.dist</li>
    <li>python313Packages.arviz.dist</li>
    <li>python313Packages.backtesting</li>
    <li>python313Packages.backtesting.dist</li>
    <li>python313Packages.bambi</li>
    <li>python313Packages.bambi.dist</li>
    <li>python313Packages.bokeh</li>
    <li>python313Packages.bokeh.dist</li>
    <li>python313Packages.bsuite</li>
    <li>python313Packages.bsuite.dist</li>
    <li>python313Packages.cltk</li>
    <li>python313Packages.cltk.dist</li>
    <li>python313Packages.coiled</li>
    <li>python313Packages.coiled.dist</li>
    <li>python313Packages.corner</li>
    <li>python313Packages.corner.dist</li>
    <li>python313Packages.fastai</li>
    <li>python313Packages.fastai.dist</li>
    <li>python313Packages.folium</li>
    <li>python313Packages.folium.dist</li>
    <li>python313Packages.geoarrow-pyarrow</li>
    <li>python313Packages.geoarrow-pyarrow.dist</li>
    <li>python313Packages.geodatasets</li>
    <li>python313Packages.geodatasets.dist</li>
    <li>python313Packages.geopandas</li>
    <li>python313Packages.geopandas.dist</li>
    <li>python313Packages.geoparquet</li>
    <li>python313Packages.geoparquet.dist</li>
    <li>python313Packages.holoviews</li>
    <li>python313Packages.holoviews.dist</li>
    <li>python313Packages.hvplot</li>
    <li>python313Packages.hvplot.dist</li>
    <li>python313Packages.inequality</li>
    <li>python313Packages.inequality.dist</li>
    <li>python313Packages.intake</li>
    <li>python313Packages.intake.dist</li>
    <li>python313Packages.k-diffusion</li>
    <li>python313Packages.k-diffusion.dist</li>
    <li>python313Packages.kokoro</li>
    <li>python313Packages.kokoro.dist</li>
    <li>python313Packages.lerobot</li>
    <li>python313Packages.lerobot.dist</li>
    <li>python313Packages.libpysal</li>
    <li>python313Packages.libpysal.dist</li>
    <li>python313Packages.lida</li>
    <li>python313Packages.lida.dist</li>
    <li>python313Packages.livelossplot</li>
    <li>python313Packages.livelossplot.dist</li>
    <li>python313Packages.llama-cloud</li>
    <li>python313Packages.llama-cloud.dist</li>
    <li>python313Packages.llama-index</li>
    <li>python313Packages.llama-index-cli</li>
    <li>python313Packages.llama-index-cli.dist</li>
    <li>python313Packages.llama-index-core</li>
    <li>python313Packages.llama-index-core.dist</li>
    <li>python313Packages.llama-index-embeddings-google-genai</li>
    <li>python313Packages.llama-index-embeddings-google-genai.dist</li>
    <li>python313Packages.llama-index-embeddings-huggingface</li>
    <li>python313Packages.llama-index-embeddings-huggingface.dist</li>
    <li>python313Packages.llama-index-embeddings-ollama</li>
    <li>python313Packages.llama-index-embeddings-ollama.dist</li>
    <li>python313Packages.llama-index-embeddings-openai</li>
    <li>python313Packages.llama-index-embeddings-openai.dist</li>
    <li>python313Packages.llama-index-graph-stores-neo4j</li>
    <li>python313Packages.llama-index-graph-stores-neo4j.dist</li>
    <li>python313Packages.llama-index-graph-stores-neptune</li>
    <li>python313Packages.llama-index-graph-stores-neptune.dist</li>
    <li>python313Packages.llama-index-legacy</li>
    <li>python313Packages.llama-index-legacy.dist</li>
    <li>python313Packages.llama-index-llms-ollama</li>
    <li>python313Packages.llama-index-llms-ollama.dist</li>
    <li>python313Packages.llama-index-llms-openai</li>
    <li>python313Packages.llama-index-llms-openai-like</li>
    <li>python313Packages.llama-index-llms-openai-like.dist</li>
    <li>python313Packages.llama-index-llms-openai.dist</li>
    <li>python313Packages.llama-index-node-parser-docling</li>
    <li>python313Packages.llama-index-node-parser-docling.dist</li>
    <li>python313Packages.llama-index-readers-database</li>
    <li>python313Packages.llama-index-readers-database.dist</li>
    <li>python313Packages.llama-index-readers-file</li>
    <li>python313Packages.llama-index-readers-file.dist</li>
    <li>python313Packages.llama-index-readers-json</li>
    <li>python313Packages.llama-index-readers-json.dist</li>
    <li>python313Packages.llama-index-readers-s3</li>
    <li>python313Packages.llama-index-readers-s3.dist</li>
    <li>python313Packages.llama-index-readers-twitter</li>
    <li>python313Packages.llama-index-readers-twitter.dist</li>
    <li>python313Packages.llama-index-readers-txtai</li>
    <li>python313Packages.llama-index-readers-txtai.dist</li>
    <li>python313Packages.llama-index-readers-weather</li>
    <li>python313Packages.llama-index-readers-weather.dist</li>
    <li>python313Packages.llama-index-vector-stores-chroma</li>
    <li>python313Packages.llama-index-vector-stores-chroma.dist</li>
    <li>python313Packages.llama-index-vector-stores-google</li>
    <li>python313Packages.llama-index-vector-stores-google.dist</li>
    <li>python313Packages.llama-index-vector-stores-milvus</li>
    <li>python313Packages.llama-index-vector-stores-milvus.dist</li>
    <li>python313Packages.llama-index-vector-stores-postgres</li>
    <li>python313Packages.llama-index-vector-stores-postgres.dist</li>
    <li>python313Packages.llama-index-vector-stores-qdrant</li>
    <li>python313Packages.llama-index-vector-stores-qdrant.dist</li>
    <li>python313Packages.llama-index.dist</li>
    <li>python313Packages.mapclassify</li>
    <li>python313Packages.mapclassify.dist</li>
    <li>python313Packages.msticpy</li>
    <li>python313Packages.msticpy.dist</li>
    <li>python313Packages.nbsmoke</li>
    <li>python313Packages.nbsmoke.dist</li>
    <li>python313Packages.nutpie</li>
    <li>python313Packages.nutpie.dist</li>
    <li>python313Packages.odc-geo</li>
    <li>python313Packages.odc-geo.dist</li>
    <li>python313Packages.odc-loader</li>
    <li>python313Packages.odc-loader.dist</li>
    <li>python313Packages.odc-stac</li>
    <li>python313Packages.odc-stac.dist</li>
    <li>python313Packages.osmnx</li>
    <li>python313Packages.osmnx.dist</li>
    <li>python313Packages.panel</li>
    <li>python313Packages.panel.dist</li>
    <li>python313Packages.plotnine</li>
    <li>python313Packages.plotnine.dist</li>
    <li>python313Packages.pymc</li>
    <li>python313Packages.pymc.dist</li>
    <li>python313Packages.shimmy</li>
    <li>python313Packages.shimmy.dist</li>
    <li>python313Packages.skrl</li>
    <li>python313Packages.skrl.dist</li>
    <li>python313Packages.spacy</li>
    <li>python313Packages.spacy-curated-transformers</li>
    <li>python313Packages.spacy-curated-transformers.dist</li>
    <li>python313Packages.spacy-loggers</li>
    <li>python313Packages.spacy-loggers.dist</li>
    <li>python313Packages.spacy-lookups-data</li>
    <li>python313Packages.spacy-lookups-data.dist</li>
    <li>python313Packages.spacy-transformers</li>
    <li>python313Packages.spacy-transformers.dist</li>
    <li>python313Packages.spacy.dist</li>
    <li>python313Packages.streamlit-folium</li>
    <li>python313Packages.streamlit-folium.dist</li>
    <li>python313Packages.textacy</li>
    <li>python313Packages.textacy.dist</li>
    <li>python313Packages.textnets</li>
    <li>python313Packages.textnets.dist</li>
    <li>python313Packages.translatehtml</li>
    <li>python313Packages.translatehtml.dist</li>
    <li>python313Packages.vmas</li>
    <li>python313Packages.vmas.dist</li>
    <li>python313Packages.wandb</li>
    <li>python313Packages.wandb.dist</li>
    <li>python313Packages.wktutils</li>
    <li>python313Packages.wktutils.dist</li>
    <li>python313Packages.xyzservices</li>
    <li>python313Packages.xyzservices.dist</li>
    <li>python314Packages.arviz-plots</li>
    <li>python314Packages.arviz-plots.dist</li>
    <li>python314Packages.backtesting</li>
    <li>python314Packages.backtesting.dist</li>
    <li>python314Packages.bokeh</li>
    <li>python314Packages.bokeh.dist</li>
    <li>python314Packages.bsuite</li>
    <li>python314Packages.bsuite.dist</li>
    <li>python314Packages.coiled</li>
    <li>python314Packages.coiled.dist</li>
    <li>python314Packages.folium</li>
    <li>python314Packages.folium.dist</li>
    <li>python314Packages.geoarrow-pyarrow</li>
    <li>python314Packages.geoarrow-pyarrow.dist</li>
    <li>python314Packages.geodatasets</li>
    <li>python314Packages.geodatasets.dist</li>
    <li>python314Packages.geopandas</li>
    <li>python314Packages.geopandas.dist</li>
    <li>python314Packages.geoparquet</li>
    <li>python314Packages.geoparquet.dist</li>
    <li>python314Packages.holoviews</li>
    <li>python314Packages.holoviews.dist</li>
    <li>python314Packages.hvplot</li>
    <li>python314Packages.hvplot.dist</li>
    <li>python314Packages.inequality</li>
    <li>python314Packages.inequality.dist</li>
    <li>python314Packages.intake</li>
    <li>python314Packages.intake.dist</li>
    <li>python314Packages.k-diffusion</li>
    <li>python314Packages.k-diffusion.dist</li>
    <li>python314Packages.libpysal</li>
    <li>python314Packages.libpysal.dist</li>
    <li>python314Packages.lida</li>
    <li>python314Packages.lida.dist</li>
    <li>python314Packages.livelossplot</li>
    <li>python314Packages.livelossplot.dist</li>
    <li>python314Packages.mapclassify</li>
    <li>python314Packages.mapclassify.dist</li>
    <li>python314Packages.msticpy</li>
    <li>python314Packages.msticpy.dist</li>
    <li>python314Packages.nbsmoke</li>
    <li>python314Packages.nbsmoke.dist</li>
    <li>python314Packages.odc-geo</li>
    <li>python314Packages.odc-geo.dist</li>
    <li>python314Packages.odc-loader</li>
    <li>python314Packages.odc-loader.dist</li>
    <li>python314Packages.odc-stac</li>
    <li>python314Packages.odc-stac.dist</li>
    <li>python314Packages.osmnx</li>
    <li>python314Packages.osmnx.dist</li>
    <li>python314Packages.panel</li>
    <li>python314Packages.panel.dist</li>
    <li>python314Packages.plotnine</li>
    <li>python314Packages.plotnine.dist</li>
    <li>python314Packages.spacy-loggers</li>
    <li>python314Packages.spacy-loggers.dist</li>
    <li>python314Packages.streamlit-folium</li>
    <li>python314Packages.streamlit-folium.dist</li>
    <li>python314Packages.wandb</li>
    <li>python314Packages.wandb.dist</li>
    <li>python314Packages.wktutils</li>
    <li>python314Packages.wktutils.dist</li>
    <li>python314Packages.xyzservices</li>
    <li>python314Packages.xyzservices.dist</li>
    <li>supercell-wx</li>
    <li>tts</li>
    <li>tts.dist</li>
  </ul>
</details>

---
<details>
<summary>Error logs: `x86_64-linux`</summary>
<details>
<summary>aider-chat-full</summary>
<pre>    
        model = Model(&quot;gpt-4-32k&quot;)
&gt;       self.assertEqual(model.info[&quot;max_input_tokens&quot;], 32 * 1024)
                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
E       KeyError: &#x27;max_input_tokens&#x27;

tests/basic/test_models.py:47: KeyError
----------------------------- Captured stdout call -----------------------------
HTTPSConnectionPool(host=&#x27;raw.githubusercontent.com&#x27;, port=443): Max retries exceeded with url: /BerriAI/litellm/main/model_prices_and_context_window.json (Caused by NameResolutionError(&quot;HTTPSConnection(host=&#x27;raw.githubusercontent.com&#x27;, port=443): Failed to resolve &#x27;raw.githubusercontent.com&#x27; ([Errno -3] Temporary failure in name resolution)&quot;))
HTTPSConnectionPool(host=&#x27;raw.githubusercontent.com&#x27;, port=443): Max retries exceeded with url: /BerriAI/litellm/main/model_prices_and_context_window.json (Caused by NameResolutionError(&quot;HTTPSConnection(host=&#x27;raw.githubusercontent.com&#x27;, port=443): Failed to resolve &#x27;raw.githubusercontent.com&#x27; ([Errno -3] Temporary failure in name resolution)&quot;))
HTTPSConnectionPool(host=&#x27;raw.githubusercontent.com&#x27;, port=443): Max retries exceeded with url: /BerriAI/litellm/main/model_prices_and_context_window.json (Caused by NameResolutionError(&quot;HTTPSConnection(host=&#x27;raw.githubusercontent.com&#x27;, port=443): Failed to resolve &#x27;raw.githubusercontent.com&#x27; ([Errno -3] Temporary failure in name resolution)&quot;))
HTTPSConnectionPool(host=&#x27;raw.githubusercontent.com&#x27;, port=443): Max retries exceeded with url: /BerriAI/litellm/main/model_prices_and_context_window.json (Caused by NameResolutionError(&quot;HTTPSConnection(host=&#x27;raw.githubusercontent.com&#x27;, port=443): Failed to resolve &#x27;raw.githubusercontent.com&#x27; ([Errno -3] Temporary failure in name resolution)&quot;))
HTTPSConnectionPool(host=&#x27;raw.githubusercontent.com&#x27;, port=443): Max retries exceeded with url: /BerriAI/litellm/main/model_prices_and_context_window.json (Caused by NameResolutionError(&quot;HTTPSConnection(host=&#x27;raw.githubusercontent.com&#x27;, port=443): Failed to resolve &#x27;raw.githubusercontent.com&#x27; ([Errno -3] Temporary failure in name resolution)&quot;))
HTTPSConnectionPool(host=&#x27;raw.githubusercontent.com&#x27;, port=443): Max retries exceeded with url: /BerriAI/litellm/main/model_prices_and_context_window.json (Caused by NameResolutionError(&quot;HTTPSConnection(host=&#x27;raw.githubusercontent.com&#x27;, port=443): Failed to resolve &#x27;raw.githubusercontent.com&#x27; ([Errno -3] Temporary failure in name resolution)&quot;))
HTTPSConnectionPool(host=&#x27;raw.githubusercontent.com&#x27;, port=443): Max retries exceeded with url: /BerriAI/litellm/main/model_prices_and_context_window.json (Caused by NameResolutionError(&quot;HTTPSConnection(host=&#x27;raw.githubusercontent.com&#x27;, port=443): Failed to resolve &#x27;raw.githubusercontent.com&#x27; ([Errno -3] Temporary failure in name resolution)&quot;))
=========================== short test summary info ============================
FAILED tests/basic/test_commands.py::TestCommands::test_cmd_read_only_with_image_file - AssertionError: 0 != 1
FAILED tests/basic/test_commands.py::TestCommands::test_cmd_tokens_output - AssertionError: False is not true
FAILED tests/basic/test_models.py::TestModels::test_max_context_tokens - KeyError: &#x27;max_input_tokens&#x27;
===== 3 failed, 460 passed, 1 skipped, 11 deselected in 100.67s (0:01:40) ======</pre>
</details>
<details>
<summary>aider-chat-with-help</summary>
<pre>    
        model = Model(&quot;gpt-4-32k&quot;)
&gt;       self.assertEqual(model.info[&quot;max_input_tokens&quot;], 32 * 1024)
                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
E       KeyError: &#x27;max_input_tokens&#x27;

tests/basic/test_models.py:47: KeyError
----------------------------- Captured stdout call -----------------------------
HTTPSConnectionPool(host=&#x27;raw.githubusercontent.com&#x27;, port=443): Max retries exceeded with url: /BerriAI/litellm/main/model_prices_and_context_window.json (Caused by NameResolutionError(&quot;HTTPSConnection(host=&#x27;raw.githubusercontent.com&#x27;, port=443): Failed to resolve &#x27;raw.githubusercontent.com&#x27; ([Errno -3] Temporary failure in name resolution)&quot;))
HTTPSConnectionPool(host=&#x27;raw.githubusercontent.com&#x27;, port=443): Max retries exceeded with url: /BerriAI/litellm/main/model_prices_and_context_window.json (Caused by NameResolutionError(&quot;HTTPSConnection(host=&#x27;raw.githubusercontent.com&#x27;, port=443): Failed to resolve &#x27;raw.githubusercontent.com&#x27; ([Errno -3] Temporary failure in name resolution)&quot;))
HTTPSConnectionPool(host=&#x27;raw.githubusercontent.com&#x27;, port=443): Max retries exceeded with url: /BerriAI/litellm/main/model_prices_and_context_window.json (Caused by NameResolutionError(&quot;HTTPSConnection(host=&#x27;raw.githubusercontent.com&#x27;, port=443): Failed to resolve &#x27;raw.githubusercontent.com&#x27; ([Errno -3] Temporary failure in name resolution)&quot;))
HTTPSConnectionPool(host=&#x27;raw.githubusercontent.com&#x27;, port=443): Max retries exceeded with url: /BerriAI/litellm/main/model_prices_and_context_window.json (Caused by NameResolutionError(&quot;HTTPSConnection(host=&#x27;raw.githubusercontent.com&#x27;, port=443): Failed to resolve &#x27;raw.githubusercontent.com&#x27; ([Errno -3] Temporary failure in name resolution)&quot;))
HTTPSConnectionPool(host=&#x27;raw.githubusercontent.com&#x27;, port=443): Max retries exceeded with url: /BerriAI/litellm/main/model_prices_and_context_window.json (Caused by NameResolutionError(&quot;HTTPSConnection(host=&#x27;raw.githubusercontent.com&#x27;, port=443): Failed to resolve &#x27;raw.githubusercontent.com&#x27; ([Errno -3] Temporary failure in name resolution)&quot;))
HTTPSConnectionPool(host=&#x27;raw.githubusercontent.com&#x27;, port=443): Max retries exceeded with url: /BerriAI/litellm/main/model_prices_and_context_window.json (Caused by NameResolutionError(&quot;HTTPSConnection(host=&#x27;raw.githubusercontent.com&#x27;, port=443): Failed to resolve &#x27;raw.githubusercontent.com&#x27; ([Errno -3] Temporary failure in name resolution)&quot;))
HTTPSConnectionPool(host=&#x27;raw.githubusercontent.com&#x27;, port=443): Max retries exceeded with url: /BerriAI/litellm/main/model_prices_and_context_window.json (Caused by NameResolutionError(&quot;HTTPSConnection(host=&#x27;raw.githubusercontent.com&#x27;, port=443): Failed to resolve &#x27;raw.githubusercontent.com&#x27; ([Errno -3] Temporary failure in name resolution)&quot;))
=========================== short test summary info ============================
FAILED tests/basic/test_commands.py::TestCommands::test_cmd_read_only_with_image_file - AssertionError: 0 != 1
FAILED tests/basic/test_commands.py::TestCommands::test_cmd_tokens_output - AssertionError: False is not true
FAILED tests/basic/test_models.py::TestModels::test_max_context_tokens - KeyError: &#x27;max_input_tokens&#x27;
===== 3 failed, 460 passed, 1 skipped, 11 deselected in 193.78s (0:03:13) ======</pre>
</details>
<details>
<summary>python313Packages.momepy</summary>
<pre>configfile: pyproject.toml
collecting ... 
collecting 0 items                                                             
collected 254 items / 1 error                                                  

==================================== ERRORS ====================================
_______________ ERROR collecting momepy/tests/test_continuity.py _______________
ImportError while importing test module &#x27;/build/source/momepy/tests/test_continuity.py&#x27;.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
/nix/store/pzdalg368npikvpq4ncz2saxnz19v53k-python3-3.13.12/lib/python3.13/importlib/__init__.py:88: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
momepy/tests/test_continuity.py:2: in &lt;module&gt;
    import neatnet
E   ModuleNotFoundError: No module named &#x27;neatnet&#x27;
=========================== short test summary info ============================
ERROR momepy/tests/test_continuity.py
!!!!!!!!!!!!!!!!!!!! Interrupted: 1 error during collection !!!!!!!!!!!!!!!!!!!!
=============================== 1 error in 2.13s ===============================</pre>
</details>
<details>
<summary>python313Packages.pandera</summary>
<pre>tests/strategies/test_strategies.py: 60 warnings
  /build/source/pandera/strategies/pandas_strategies.py:759: HypothesisWarning: bool(from_dtype(dtype(&#x27;int64&#x27;), allow_nan=False, allow_infinity=False)) is always True, did you mean to draw a value?
    if strategy:

tests/strategies/test_strategies.py: 840 warnings
  /build/source/tests/strategies/test_strategies.py:759: FutureWarning: The behavior of DataFrame.sum with axis=None is deprecated, in a future version this will reduce over both axes and return a scalar. To retain the old behavior, pass axis=0 (or do not pass axis)
    assert example.isna().sum(axis=None).item() &gt;= 0

tests/strategies/test_strategies.py: 767 warnings
  /build/source/tests/strategies/test_strategies.py:900: FutureWarning: DataFrame.applymap has been deprecated. Use DataFrame.map instead.
    return pandas_obj.applymap(lambda x: min_val &lt;= x &lt;= max_val)

-- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
=========================== short test summary info ============================
FAILED tests/io/test_pandas_io.py::test_frictionless_schema_parses_correctly[frictionless_schema0] - ImportError: Frictionless schema parsing requires &#x27;frictionless&#x27; to be installed. Please install &#x27;frictionless&#x27; to use this feature.
FAILED tests/io/test_pandas_io.py::test_frictionless_schema_parses_correctly[frictionless_schema1] - ImportError: Frictionless schema parsing requires &#x27;frictionless&#x27; to be installed. Please install &#x27;frictionless&#x27; to use this feature.
FAILED tests/io/test_pandas_io.py::test_frictionless_schema_primary_key[frictionless_schema0] - ImportError: Frictionless schema parsing requires &#x27;frictionless&#x27; to be installed. Please install &#x27;frictionless&#x27; to use this feature.
FAILED tests/io/test_pandas_io.py::test_frictionless_schema_primary_key[frictionless_schema1] - ImportError: Frictionless schema parsing requires &#x27;frictionless&#x27; to be installed. Please install &#x27;frictionless&#x27; to use this feature.
FAILED tests/io/test_pandas_io.py::test_frictionless_schema_with_description_and_title[frictionless_schema0] - ImportError: Frictionless schema parsing requires &#x27;frictionless&#x27; to be installed. Please install &#x27;frictionless&#x27; to use this feature.
= 5 failed, 4088 passed, 4 skipped, 22 deselected, 29 xfailed, 8402 warnings in 254.10s (0:04:14) =</pre>
</details>
<details>
<summary>python314Packages.momepy</summary>
<pre>configfile: pyproject.toml
collecting ... 
collecting 0 items                                                             
collected 254 items / 1 error                                                  

==================================== ERRORS ====================================
_______________ ERROR collecting momepy/tests/test_continuity.py _______________
ImportError while importing test module &#x27;/build/source/momepy/tests/test_continuity.py&#x27;.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
/nix/store/qr0wqh2damb6asnyz4jkx61id5yg3qh2-python3-3.14.3/lib/python3.14/importlib/__init__.py:88: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
momepy/tests/test_continuity.py:2: in &lt;module&gt;
    import neatnet
E   ModuleNotFoundError: No module named &#x27;neatnet&#x27;
=========================== short test summary info ============================
ERROR momepy/tests/test_continuity.py
!!!!!!!!!!!!!!!!!!!! Interrupted: 1 error during collection !!!!!!!!!!!!!!!!!!!!
=============================== 1 error in 2.12s ===============================</pre>
</details>
<details>
<summary>python314Packages.pandera</summary>
<pre>tests/strategies/test_strategies.py: 60 warnings
  /build/source/pandera/strategies/pandas_strategies.py:759: HypothesisWarning: bool(from_dtype(dtype(&#x27;int64&#x27;), allow_nan=False, allow_infinity=False)) is always True, did you mean to draw a value?
    if strategy:

tests/strategies/test_strategies.py: 840 warnings
  /build/source/tests/strategies/test_strategies.py:759: FutureWarning: The behavior of DataFrame.sum with axis=None is deprecated, in a future version this will reduce over both axes and return a scalar. To retain the old behavior, pass axis=0 (or do not pass axis)
    assert example.isna().sum(axis=None).item() &gt;= 0

tests/strategies/test_strategies.py: 767 warnings
  /build/source/tests/strategies/test_strategies.py:900: FutureWarning: DataFrame.applymap has been deprecated. Use DataFrame.map instead.
    return pandas_obj.applymap(lambda x: min_val &lt;= x &lt;= max_val)

-- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
=========================== short test summary info ============================
FAILED tests/io/test_pandas_io.py::test_frictionless_schema_parses_correctly[frictionless_schema0] - ImportError: Frictionless schema parsing requires &#x27;frictionless&#x27; to be installed. Please install &#x27;frictionless&#x27; to use this feature.
FAILED tests/io/test_pandas_io.py::test_frictionless_schema_parses_correctly[frictionless_schema1] - ImportError: Frictionless schema parsing requires &#x27;frictionless&#x27; to be installed. Please install &#x27;frictionless&#x27; to use this feature.
FAILED tests/io/test_pandas_io.py::test_frictionless_schema_primary_key[frictionless_schema0] - ImportError: Frictionless schema parsing requires &#x27;frictionless&#x27; to be installed. Please install &#x27;frictionless&#x27; to use this feature.
FAILED tests/io/test_pandas_io.py::test_frictionless_schema_primary_key[frictionless_schema1] - ImportError: Frictionless schema parsing requires &#x27;frictionless&#x27; to be installed. Please install &#x27;frictionless&#x27; to use this feature.
FAILED tests/io/test_pandas_io.py::test_frictionless_schema_with_description_and_title[frictionless_schema0] - ImportError: Frictionless schema parsing requires &#x27;frictionless&#x27; to be installed. Please install &#x27;frictionless&#x27; to use this feature.
= 5 failed, 4088 passed, 4 skipped, 22 deselected, 29 xfailed, 8400 warnings in 272.26s (0:04:32) =</pre>
</details>
</details>

--- a/pkgs/development/python-modules/xyzservices/default.nix
+++ b/pkgs/development/python-modules/xyzservices/default.nix
@@ -11,12 +11,12 @@
 
 buildPythonPackage rec {
   pname = "xyzservices";
-  version = "2025.11.0";
+  version = "2026.3.0";
   pyproject = true;
 
   src = fetchPypi {
     inherit pname version;
-    hash = "sha256-L8crSVArJQI/1x6PUy+0vt278KoSTZDqJdukT1ReF84=";
+    hash = "sha256-0iaGal2On+8zcDTY2jeoKY8KHZ0UibQBjmlXnrMh/qQ=";
   };
 
   nativeBuildInputs = [

